File: PostServiceBackEnd/PostServiceBackEnd/Entities/Parcel.cs
namespace PostServiceBackEnd.Entities
{
    public class Parcel
    {
        public int Id { get; set; }
        public int Weight { get; set; }
        public string Phone { get; set; }
        public string Description { get; set; }
        public int? ParcelMachineId { get; set; }
        public ParcelMachine ParcelMachine { get; set; }
    }
}

File: PostServiceFrontEnd/src/app/Models/parcel.ts
import { ParcelMachine } from './parcelMachine';

export interface Parcel {
  id?: number;
  receiverFullName: string;
  weight: string;
  phone: string;
  description: string;
  parcelMachineId: string;
  parcelMachine?: ParcelMachine;
}
